### PR TITLE
Merge cmdline and JSON object palette

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

**Comment by Gymnasiast** [comment]
Not sure why win32 fails to compile. All the required headers appear to be included.

**Comment by duncanspumpkin** [comment]
Think its just a mistake in that one run of the build. I've told it to rerun.

File: src/openrct2/CmdlineSprite.cpp [hunk 1]
```diff
@@ -236,7 +215,7 @@ static bool sprite_file_export(int32_t spriteIndex, const char* outPath)
     dpi.pitch = 0;
     dpi.zoom_level = 0;
 
-    std::memcpy(spriteFilePalette, CmdlineSprite::_standardPalette, 256 * 4);
+    std::memcpy(spriteFilePalette, StandardPalette, 256 * sizeof(PaletteBGRA));
```
Comment (IntelOrca) [inline src/openrct2/CmdlineSprite.cpp:218]: `spriteFilePalette = StandardPalette`

**Comment by IntelOrca** [comment]
Just the one suggestion. I think it would be nice to make a struct called GamePalette or something and expose that rather than an array. But up to you.

**Comment by Gymnasiast** [comment]
I have now made the requested changes.

I have tested it with the following:
- Building g2.dat
- Extracting an image from g2.dat
- Reading PNG files from a JSON object (including remap colours)

**Pull request merged**